packages/contracts-periphery/contracts/L1/L1ERC721Bridge.sol

```diff
@@ -188,20 +212,81 @@ contract L1ERC721Bridge is Semver, CrossDomainEnabled, Initializable {
+                // slither-disable-next-line reentrancy-events
+                emit ERC721BridgeFinalized(
+                    _localToken,
+                    _remoteToken,
+                    _from,
+                    _to,
+                    _tokenId,
+                    _extraData
+                );
+            }
+        } catch {
+            // If the token ID for this L1/L2 NFT pair is not escrowed in the L1 Bridge or if
+            // another error occurred during finalization, we initiate a cross-domain message to
+            // send the NFT back to its original owner on L2. This can happen if an L2 native NFT is
+            // bridged to L1, or if a user mistakenly entered an incorrect L1 ERC721 address.
+            bytes memory message = abi.encodeWithSelector(
+                L2ERC721Bridge.finalizeBridgeERC721.selector,
+                _remoteToken,
+                _localToken,
+                address(this), // Set the new _from address to be this contract since the NFT was
+                // never transferred to the recipient on this chain.
+                _from, // Refund the NFT to the original owner on the remote chain.
+                _tokenId,
+                _extraData
+            );
+
+            // Send the message to the L2 bridge.
+            // slither-disable-next-line reentrancy-events
+            sendCrossDomainMessage(otherBridge, 600_000, message);
+
+            // slither-disable-next-line reentrancy-events
+            emit ERC721BridgeFailed(_localToken, _remoteToken, _from, _to, _tokenId, _extraData);
+        }
+    }
+
+    /**
+     * @notice Completes an outbound token transfer. Public function, but can only be called by
+     *         this contract. It's security critical that there be absolutely no way for anyone to
+     *         trigger this function, except by explicit trigger within this contract. Used as a
+     *         simple way to be able to try/catch any type of revert that could occur during an
+     *         ERC721 mint/transfer.
+     *
+     * @param _localToken  Address of the ERC721 on this chain.
+     * @param _remoteToken Address of the corresponding token on the remote chain.
+     * @param _to          Address of the receiver.
+     * @param _tokenId     ID of the token being deposited.
+     */
+    function completeOutboundTransfer(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _tokenId
+    ) external onlySelf {
+        // Checks that the L1/L2 NFT pair has a token ID that is escrowed in the L1 Bridge. Without
+        // this check, an attacker could steal a legitimate L1 NFT by supplying an arbitrary L2 NFT
+        // that maps to the L1 NFT.
         require(
             deposits[_localToken][_remoteToken][_tokenId] == true,
-            "Token ID is not escrowed in the L1 Bridge"
+            "L1ERC721Bridge: token ID is not escrowed in l1 bridge for this l1/l2 nft pair"
```

Comment:
Any reason as to not just prepend `L1ERC721Bridge` to the previous error message? Also prefer caps for L1 and L2

Reply:
Fixed